index.html

```diff
@@ -1,13 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-   
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Onomeh's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins:regular,bold,500,600&amp;display=swap"
+        />
+      <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+        <header class="toolbar-header">
+        <div class="tools">
+          <a href="/" class="logo">Onomeh</a>
+            <button class="button" type="button">
+              <img src="../images/Icons/Union.png" alt="more" />
+            </button>
```

Comment:
- [ ] Instead of using the `button` tag for adding hamburger, let's use the `nav` tag & this hamburger icon should be inside nav.
- You can use [Font Awesome](https://fontawesome.com/) website also or you can use images also for the hamburger icon.
- [ ] The hamburger icon is not showing, there is some error in the image source link, Let's fix this.